.drone.yml

```diff
@@ -36,7 +36,7 @@ steps:
   - name: sonar scanner
     image: quay.io/ukhomeofficedigital/sonar-scanner:v3.0.2
     depends_on:
-      - build project
+      - test project
```

Comment:
This changes it to run in serial to the test step instead of parallel with it -- is that intentional?